### PR TITLE
[ADD][15.0] passport_expiration

### PR DESCRIPTION
**Pull request by yankinmax**
Supersedes:
- [ ] https://github.com/OCA/vertical-travel/pull/60

### CODE REVIEW

**Review by leemannd — APPROVED** [review]

**Comment by leemannd** [comment]
Technically the code is good, but we loos the git history. I think this should be respected

**Comment by yankinmax** [comment]
@leemannd IDK how to transfer git commit through different submodules. That's why did it such way.

**Comment by OpenCode** [comment]
> @leemannd IDK how to transfer git commit through different submodules. That's why did it such way.

You can follow this guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0

**Comment by leemannd** [comment]
@yankinmax It should work:
- take your base PR https://github.com/OCA/vertical-travel/pull/60
- add a remote linked to this repository
- push the result to this repository
Don't forget to squash administrative and translations commits

**Comment by yankinmax** [comment]
@leemannd is ok now?

**Review by leemannd — APPROVED** [review]

**Comment by leemannd** [comment]
Hello @damdam-s , a review is welcome

**Comment by leemannd** [comment]
Hello @OCA/human-resources-maintainers a review and merge is welcome

**Review by damdam-s — APPROVED** [review]
lgtm 🚀

File: passport_expiration/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2013 Savoir-faire Linux
+# Copyright 2021 Camptocamp SA
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Passport Expiration",
+    "version": "15.0.0.1.0",
```
Comment (Saran440) [inline passport_expiration/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

I have a little comment

Code review, ok :+1:

**Comment by giarve** [comment]
For all people interested, there is the migration for 16.0 with a renamed module name https://github.com/OCA/hr/pull/1468 with additional expiration date for ID document (not only passport)

**Pull request merged**